src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -118,7 +129,15 @@ class ExplorerButtonGroup extends React.Component {
       filter.data_format = this.props.filter.data_format;
     }
     let resultManifest = await this.props.downloadRawDataByTypeAndFilter(
-      resourceType, filter, [refFieldInResourceIndex, resourceFieldInResourceIndex],
+      resourceType,
+      filter,
+      [
+        refFieldInResourceIndex,
+        resourceFieldInResourceIndex,
+        fileNameField,
+        md5Field,
+        fileSizeField,
+      ],
     );
     resultManifest = resultManifest.filter(
       x => !!x[resourceFieldInResourceIndex],
```

Comment:
you probably want also to change logics in here so the manifest downloaded from data tab will also has these new fields

Reply:
Gotchu, I missed this -- Could you tell me what this line is doing? I thought it was removing all elements from the manifest that had a blank `resourceFieldInResourceIndex` property.

Reply:
wait... actually you are right. I was getting a manifest without new fields, but this line is irrlevant. Sorry for the false alarm. I'll make this as resolved 🤦‍♂

Reply:
No worries, thank you for looking over my code in such detail!